frontend/src/components/data-provider/AddDataSection.tsx

```diff
@@ -12,7 +13,9 @@ export default () => {
           individuals and organisations to contribute their valuable data to
           drive insights and innovations in sustainable transportation.
         </p>
-        <Button className="bg-[#006064] px-6 max-w-fit py-3.5">Add data</Button>
+        <Button className="max-w-fit bg-[#006064] px-6 py-3.5">
```

Comment:
have to set asChild=true to the component so it renders button properties on its first child, which is the <Link>